### PR TITLE
update docs on pinned indication

### PR DESCRIPTION
**Pull request by pschuegr**
Let me know if you think I should have worded anything differently.

Cheers!

### CODE REVIEW

**Comment by dozius** [comment]
Sorry for the late reply. I've been busy with other projects in my spare time.

I found this to be a little too wordy. Here is my suggested edit.

```
When a track or device is pinned, the MIDI Fighter Twister will not follow the current track or device selected with the mouse or another controller. Info about the state of a pinned track or device is available in the Studio I/O panel in Bitwig Studio.
```

**Comment by pschuegr** [comment]
Updated with your wording.

**Comment by dozius** [comment]
Thanks!
